XCUITests/FxScreenGraph.swift

```diff
@@ -99,11 +100,18 @@ func createScreenGraph(_ app: XCUIApplication, url: String = "https://www.mozill
             }
 
             scene.tap(startBrowsingButton, to: NewTabScreen)
+            scene.tap(app.buttons["Sign in to Firefox"], to: FxSyncLogin)
         }
 
         i += 1
     }
 
+    map.createScene(FxSyncLogin) { scene in
+        scene.backAction = {
+            app.buttons["Cancel"].tap()
```

Comment:
Yeah, actually I had the same doubt when implementing the test...If tapping on Cancel takes you to a different place depending on from where you got to FxSyncLogin maybe it is not possible to add it to FxScreenGraph as a general action...

@jhugman, you said three options, but there are only two, what is the third one ;) ? I do not really know which one would be the best, and if you do not like any of the approaches, maybe it is better to keep that as part of the test not the FxScreenGraph?